tests/psa-client-server/psasim/tools/psa_autogen.py

```diff
@@ -32,14 +35,14 @@
             irqs = []
 
         try:
-            os.mkdir("psa_manifest")
+            os.mkdir(GENERATED_H_PATH)
             print("Generating psa_manifest directory")
         except OSError:
             print ("PSA manifest directory already exists")
 
-        man  = open(str("psa_manifest/" + FILENAME + ".h"), "w")
-        pids = open("psa_manifest/pid.h", "a")
-        sids = open("psa_manifest/sid.h", "a")
+        man  = open(os.path.join(GENERATED_H_PATH, FILENAME + ".h"), "w")
+        pids = open(os.path.join(GENERATED_H_PATH, "pid.h"), "a")
+        sids = open(os.path.join(GENERATED_H_PATH, "sid.h"), "a")
```

Comment:
question: would it make sense to create a function `open_in_include_dir()` to avoid the repetition of the `os.path.join`s?